template.yaml

```diff
@@ -445,6 +470,51 @@ Resources:
         SecurityGroupIds: !Split [ "," , !GetAtt LambdaSecurityGroup.GroupId ]
         SubnetIds: !Split [ "," , !Ref Subnets ]
 
+  CheckRegionSupportSnapshotExportToS3LambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub ${ProjectName}-check-region-support-snapshot-export-to-S3-sf
+      Description: Lambda functions required to check whether region supports RDS snapshot export to s3.
+      Role: !GetAtt LambdaExecutionRole.Arn
+      Runtime: python3.7
+      CodeUri: src/export/copy_across_regions/
+      Handler: check_region_support_snapshot_export_to_s3_function.lambda_check_region_support_snapshot_export_to_s3
+      MemorySize: 128
+      Layers:
+        - !Ref CommonLambdaLayer
+      Timeout: 900
+      Environment:
+        Variables:
+          ExportSnapshotSupportedRegion: !Ref ExportSnapshotSupportedRegion
+      VpcConfig:
+        !If [UseVPCAndSubnetsCondition, {
+          SecurityGroupIds: !Split [ "," , !GetAtt LambdaSecurityGroup.GroupId ] ,
+          SubnetIds: !Split [ "," , !Ref Subnets ]
+        } ,  !Ref "AWS::NoValue"  ]
+
+  SnapshotCopyToExportToS3SupportedRegionLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub ${ProjectName}-snapshot-copy-to-export-to-S3-supported-region-sf
+      Description: Lambda functions required to copy RDS snapshot to export to s3 supported region.
+      Role: !GetAtt LambdaExecutionRole.Arn
+      Runtime: python3.7
+      CodeUri: src/export/copy_across_regions/
+      Handler: copy_snapshot_to_s3_export_supportable_region_function.lambda_copy_snapshot_to_s3_export_supportable_region
```

Comment:
more concise naming

Reply:
@namitad I am all for being as concise as possible to the point where you don't loose information. The result is what you see. I don't believe it can be shorter. Can you give me an example without loosing the essence? Otherwise let's keep it as it is.

Reply:
@despot considering that the method basically does copy of a snapshot from one region to another, can the naming be kept generic  such as `copy_snapshot_to_target_region_function` and `lambda_copy_snapshot_to_target_region`

Reply:
@namitad  
"from one region to another" 
The boto3 call is only generic. The method copies to the 'ExportSnapshotSupportedRegion' only. Changing it to your proposal misinterprets the method. I wouldn't like people to place non-supportable regions in 'ExportSnapshotSupportedRegion' so they can use it as generic or place logic in the method that is not about exporting to the supportable region. Once a generic use case appears, the appropriate generic method can be created. For now, I believe, the name you suggested is taking us to a wrong direction.

Do you want me to change 
copy_snapshot_to_s3_export_supportable_region_function
to 
copy_snapshot_to_region_that_supports_export_to_s3_function
? 
Otherwise let's please leave it as is.